### PR TITLE
PostgreSQL

### PR DESCRIPTION
**Pull request by wyatt-troia**
@mjones61692: could you please take a look at these changes?

[Ticket: Develop Postgres data seeding script for 10M records](https://trello.com/c/SIZm26P2/4-develop-postgres-data-seeding-script-for-10m-records)

### CODE REVIEW

File: database/seed.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+      days.push(`${monthToYear[month]}-${monthString}-${dayString}`)
+    }
+  }
+  // console.log(days);
+  for (let i = 0; i < 1000; i++) {
+    let bookedDatesBatch = [];
+    for (let j = 0; j < 10000; j++) {
+      let minStay = await knex.select('minstay').from('bookings.listings').where('id', (j + 1) + i * 10000).limit(1);
+      minstay = minStay[0]['minstay'];
+      debugger;
```
Comment (mjones61692-zz) [inline database/seed.js]: I don't know if you are still debugging this code, but if it is working, don't forget to remove debugger's

File: package.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "react-dev": "webpack -d --watch",
     "server-dev": "nodemon server/index.js",
-    "populate-db": "createdb topbunk && psql topbunk < database/init.sql",
+    "init-db": "createdb topbunk && psql topbunk < database/init.sql",
```
Comment (mjones61692-zz) [inline package.json:9]: I like this name change!

**Review by mjones61692-zz — APPROVED** [review]
@wyatt-troia, I made a couple of comments, but overall everything looks great. I especially like the addition of the comments. They make your code much more readable. Feel free to merge this.

File: database/seed.js (hunk 1 above)
Comment (wyatt-troia) [reply database/seed.js]: Removed.

**Comment by wyatt-troia** [comment]
Thanks, @mjones61692 !

**Pull request merged**